examples/c++-api/count-stuff.cc

```diff
@@ -0,0 +1,38 @@
+// A demonstration of using khmer to count k-mers; in this case, the request
+// was for exact counting.
+
+#include <vector>
+#include <cmath>
+#include "khmer.hh"
+#include "counting.hh"
+
+using namespace khmer;
+
+int main()
+{
+    unsigned int ksize = 11;
```

Comment:
Might want to mention that ksize should be odd; if it's even, then reverse complements will collapse into one entry, and you'll want to use a tablesize of ... what, 4**(k-1) + k, or something?